src/components/NoteDetails/NoteDetailsTopBar.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
```

Comment:
This code is the same as in another pull request: https://github.com/DarthVanger/NeuralNotes/pull/216/files.

This code is not for search and shouldn't be in this pull request. Please remove it.